From 228c4cd495f289d95c263be99751d35992a362ae Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sat, 18 Apr 2020 11:32:54 +0300
Subject: [PATCH] [SW-0] goya2: remove sbtool from arc

It is now moved to arc_gnu where it is used by post-image script

Change-Id: I51c0dd1a636c03b1e33535cbcd5789c9ff1ef43c
---

diff --git a/configs/goya2_arc_defconfig b/configs/goya2_arc_defconfig
index 09a8a71..e82c571 100644
--- a/configs/goya2_arc_defconfig
+++ b/configs/goya2_arc_defconfig
@@ -20,4 +20,3 @@
 # BR2_PACKAGE_IFUPDOWN_SCRIPTS is not set
 # BR2_TARGET_ROOTFS_TAR is not set
 BR2_PACKAGE_ARCBOOT=y
-BR2_PACKAGE_HOST_ARCBOOT=y
